### PR TITLE
tracking first template app

### PR DESCRIPTION
**Pull request by calellowitz**
@dimagi/product  @NoahCarnahan 
final guided tour analytic property

### CODE REVIEW

File: corehq/apps/app_manager/views/apps.py [hunk 1]
```diff
@@ -69,8 +69,9 @@
 from corehq.apps.app_manager.decorators import no_conflict_require_POST, \
     require_can_edit_apps, require_deploy_apps
 from django_prbac.utils import has_privilege
-from corehq.apps.analytics.tasks import track_app_from_template_on_hubspot
+from corehq.apps.analytics.tasks import track_app_from_template_on_hubspot, update_kissmetrics_properties
 from corehq.apps.analytics.utils import get_meta
+from corehq.apps.tour import tours
```
Comment (dimagimon) [inline corehq/apps/app_manager/views/apps.py]: F811 redefinition of unused 'tours' from line 28

File: corehq/apps/app_manager/views/apps.py [hunk 2]
```diff
@@ -114,6 +114,8 @@ def default_new_app(request, domain):
     """
     meta = get_meta(request)
     track_app_from_template_on_hubspot.delay(request.couch_user, request.COOKIES, meta)
+    if tours.NEW_APP.is_enabled(request.user):
+        update_kissmetrics_properties(request.couch_user.username, {'First Template App Chosen': 'blank'})
```
Comment (NoahCarnahan) [inline corehq/apps/app_manager/views/apps.py]: You want to use `delay()` here I think

File: corehq/apps/app_manager/views/apps.py [hunk 3]
```diff
@@ -297,6 +299,8 @@ def copy_app(request, domain):
 def app_from_template(request, domain, slug):
     meta = get_meta(request)
     track_app_from_template_on_hubspot.delay(request.couch_user, request.COOKIES, meta)
+    if tours.NEW_APP.is_enabled(request.user):
+        update_kissmetrics_properties(request.couch_user.username, {'First Template App Chosen': '%s' % slug})
```
Comment (NoahCarnahan) [inline corehq/apps/app_manager/views/apps.py]: and here as well

File: corehq/apps/app_manager/views/apps.py [hunk 4]
```diff
@@ -297,6 +299,8 @@ def copy_app(request, domain):
 def app_from_template(request, domain, slug):
     meta = get_meta(request)
     track_app_from_template_on_hubspot.delay(request.couch_user, request.COOKIES, meta)
+    if tours.NEW_APP.is_enabled(request.user):
+        update_kissmetrics_properties.delay(request.couch_user.username, {'First Template App Chosen': '%s' % slug})
```
Comment (dimagimon) [inline corehq/apps/app_manager/views/apps.py]: E501 line too long (116 > 115 characters)

**Pull request merged**